src/components/CardBack.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react'
+
+export default class CardBack extends Component {
+
+  imageSource() {
+    return `https://mkleincreative.files.wordpress.com/2016/09/cardback1.jpg`
+  }
+
+
+  render() {
+    return (
+      <div className="Card">
```

Comment:
Shouldn't this class be `CardBack` and not `Card`. You have another component called `Card` so its the only component that should use the `Card` classname